services/workflows-service/src/note/note.model.ts

```diff
@@ -0,0 +1,60 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { EntityType, Noteable } from '@prisma/client';
+import { IsString } from 'class-validator';
+
+export class NoteModel {
+  @ApiProperty({
+    required: true,
+    type: String,
+  })
+  @IsString()
+  id!: string;
+
+  @ApiProperty({
+    required: true,
+    type: String,
+  })
+  @IsString()
+  entityId!: string;
+
+  @ApiProperty({
+    required: true,
+    enum: ['Business', 'EndUser'],
+  })
+  @IsString()
```

Comment:
Don't entirely understand if we use the validation. If not then we'd remove the validation decarotors, if yes there are decorators like IsOneOf or some other one for enum.